x/programs/rust/wasmlanche_sdk/src/store/mod.rs

```diff
@@ -69,18 +78,27 @@ impl Context {
             Ok(bytes) => bytes,
             Err(e) => return StoreResult::Err(e),
         };
-        match store_key_value(self, key_bytes, value) {
+
+        match store_key_value(self, &key_bytes, value) {
             Ok(_) => StoreResult::Ok(self),
             Err(e) => StoreResult::Err(e),
         }
     }
-    pub fn get_value<T>(&self, name: &str) -> Result<T, StorageError>
+
+    /// Returns the value of the field `name` from the host.
+    /// # Errors
+    /// Returns a `StorageError` if the field does not exist or if the bytes are invalid.
```

Comment:
```suggestion
```

yep these are old

Reply:
One of the `clippy::pedantic` lints is making sure that there's an `# Errors` section for any function that errors, so we need this. I just don't know how accurate the comment is.